app/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8">
     <link rel="icon" type="image/png" href="./images/favicon.ico"/>
-    <title>{{ 'label.heading.mifosxclient' | translate }}</title>
```

Comment:
This is not a correct fix!, the labels are supposed to be used for translation

Reply:
Ok.
The onload function might be the solution to this issue.